internal/xds/translator/testdata/in/xds-ir/tcp-endpoint-stats.yaml

```diff
@@ -5,11 +5,12 @@ tcp:
 - name: "tcp-route-enable-endpoint-stats"
   address: "0.0.0.0"
   port: 10080
-  destination:
-    name: "tcp-route-simple-dest"
-    settings:
-    - endpoints:
-      - host: "1.2.3.4"
-        port: 50000
-      - host: "5.6.7.8"
-        port: 50001
+  routes:
```

Comment:
This is a bug in the tcp-endpoint-stats test. The TCPRoute in the TCP Listener ir is incorrect.